app/client/widgets/AttachmentsWidget.ts

```diff
@@ -27,6 +30,8 @@ export class AttachmentsWidget extends NewAbstractWidget {
 
   private _attachmentsTable: MetaTableData<'_grist_Attachments'>;
   private _height: KoSaveableObservable<string>;
+  private _uploadingTimeouts: Partial<Record<UIRowId, number>> = {};
```

Comment:
Might be a good idea to use the `onDispose` hook in `AttachmentsWidget` and clear any timeouts that may be left in here.